frontend/src/packages/trucks/libs/types/truck-form-model-type/truck-form-model.type.ts

```diff
@@ -0,0 +1,24 @@
+import { type ValueOf } from '~/libs/types/types.js';
+
+import {
+  type TruckManufacturer,
+  type TruckTowType,
+} from '../../enums/enums.js';
+
+type TruckFormModel = {
+  manufacturer: {
+    label: string;
+    value: ValueOf<typeof TruckManufacturer>;
+  };
```

Comment:
maybe we can re-use SelectOption here? (make it generic)

Reply:
like this:
type TruckFormModel = {
  manufacturer: SelectOption<ValueOf<typeof TruckManufacturer>>;
  capacity: number;
  pricePerKm: number;
  licensePlateNumber: string;
  year: number;
  towType: SelectOption<ValueOf<typeof TruckTowType>>;
  drivers: string;
};